From c67d7851cd95eea24139a3a9759ea68d9bcfc72a Mon Sep 17 00:00:00 2001
From: Igor Grinberg <grinberg@compulab.co.il>
Date: Sun, 3 Mar 2013 13:19:19 +0200
Subject: ARM: i.MX6: cm-fx6: remove LDB backlight GPIO

cm-fx6 does not use the LDB backlight GPIO, so remove it.

Signed-off-by: Igor Grinberg <grinberg@compulab.co.il>
---
 arch/arm/mach-mx6/board-cm-fx6.c |    3 ---
 1 file changed, 3 deletions(-)

diff --git a/arch/arm/mach-mx6/board-cm-fx6.c b/arch/arm/mach-mx6/board-cm-fx6.c
index 6e8ac76..587fd0e 100644
--- a/arch/arm/mach-mx6/board-cm-fx6.c
+++ b/arch/arm/mach-mx6/board-cm-fx6.c
@@ -96,7 +96,6 @@
 #define SB_FX6_GPIO_EXT_BASE		IMX_GPIO_NR(8, 0)
 #define SB_FX6_LCD_RST			IMX_GPIO_NR(8, 11)
 
-#define MX6_ARM2_LDB_BACKLIGHT		IMX_GPIO_NR(1, 9)
 #define MX6_ARM2_USB_OTG_PWR		IMX_GPIO_NR(3, 22)
 #define MX6_ARM2_CAN2_EN		IMX_GPIO_NR(5, 24)
 #define MX6_ARM2_CAN1_STBY		IMX_GPIO_NR(7, 12)
@@ -1189,8 +1188,6 @@ static void __init cm_fx6_init(void)
 	imx_asrc_data.asrc_audio_clk = clk_get(NULL, "asrc_serial_clk");
 	imx6q_add_asrc(&imx_asrc_data);
 
-	gpio_request_one(MX6_ARM2_LDB_BACKLIGHT, GPIOF_OUT_INIT_HIGH,
-			 "ldb-backlight");
 	imx6q_add_otp();
 	imx6q_add_viim();
 	imx6q_add_imx2_wdt(0, NULL);
-- 
1.7.9.5

